packages/core/src/AbstractRenderer.ts

```diff
@@ -386,6 +387,15 @@ export abstract class AbstractRenderer extends EventEmitter
         return renderTexture;
     }
 
+    /**
+     * Adds a new system to the renderer.
+     *
+     * @param ClassRef - Class reference
+     * @param [name] - Property name for system
```

Comment:
We don't need [name] here you can just use name. Our docs infer optional.

Reply:
Fixed. I misunderstand what a `[]` is in JSDoc.